### ISSUE TITLE
Use new show table helper's form builder in all show pages

### ISSUE THREAD

**Issue by prioux** [aces/cbrain#754]
Issue #743 discussed how we could make our `show table helpers` provide the Rails Form Builder handle to the render blocks.

PR #744 implemented this feature, and at the same time it used the new feature on one such page, the `show` page for `users`.

Now it's time to apply this everywhere we have a `show` page:

```
-rw-r--r--  1 prioux  staff   5077 Jul 13  2017 access_profiles/show.html.erb
-rw-r--r--  1 prioux  staff  24519 Jul 13  2017 bourreaux/show.html.erb
-rw-r--r--  1 prioux  staff  11866 Jul  4 16:09 data_providers/show.html.erb
-rw-r--r--  1 prioux  staff   2859 Jul  4 16:26 exception_logs/show.html.erb
-rw-r--r--  1 prioux  staff   7293 Mar  9 11:08 groups/show.html.erb
-rw-r--r--  1 prioux  staff   1670 Feb 27  2017 help_documents/show.html.erb
-rw-r--r--  1 prioux  staff   4679 Jul 13  2017 signups/show.html.erb
-rw-r--r--  1 prioux  staff   5386 Jul 13  2017 sites/show.html.erb
-rw-r--r--  1 prioux  staff  12872 Mar  9 11:08 tasks/show.html.erb
-rw-r--r--  1 prioux  staff   3479 Jul 13  2017 tool_configs/show.html.erb
-rw-r--r--  1 prioux  staff  12783 Jul  4 16:09 userfiles/show.html.erb
-rw-r--r--  1 prioux  staff   9511 Jul  5 16:57 users/show.html.erb
```

It is important to remember that unlike in the previous implementation of the show table helpers, we have to explicitly pass the `:edit_condition => true` for the show tables that are editable.

**Comment by MontrealSergiy** [comment]
Sometimes old school *_tags helpers are kept. Namely for the cases of divergent html and attribute name, subatrributes and checkboxes were not handled, as these might require more profound and risky changes. 

Theoretically check_box should work,
https://stackoverflow.com/questions/18684629/rails-4-checkbox-and-boolean-field
yet it seems would require controller changes or model attribute aliases. providers/data_providers file is not addressed at all (model and html name are different). I am not so sure would a comlex change justifies the style improvements.

**Comment by prioux** [comment]
Just update the input fields that are straightforward to change.

**Comment by MontrealSergiy** [comment]
actually checkboxes data_providers seems started working after yesterday's fix

**Comment by prioux** [comment]
Closed by #755
